views/index.hbs

```diff
@@ -1,4 +1,4 @@
-<html lang="en">
+it<html lang="en">
```

Comment:
i think you accidentally wrote something here bini ... what's the "it" on the first line?

Reply:
it is written by mistake